### DISCUSSION TITLE
"Unrecognised setting unsteady" error on linear_horten example code

### DISCUSSION THREAD

**Discussion by koheinogi** [ImperialCollegeLondon/sharpy#198] [Q&A] [answered]
When I ran one of the example codes named linear_horten without any modification, 

> Exception: [NotRecognisedSetting('Unrecognised setting unsteady. Please check input file and/or `documentation')]`

this error came out.
Do you have any idea to fix this problem?
<img width="801" alt="Screenshot 2022-06-07 at 18 07 08" src="https://user-images.githubusercontent.com/57959743/172441599-77547b24-3f6a-405b-8aad-4e43ee3495cc.png">

**Comment by ngoiz** [comment, marked as answer]
Hi @koheinogi,

Thanks for raising this. This error was introduced in the latest release of SHARPy to avoid deprecated/unrecognised settings in the simulation. 

Removing it should solve the issue.

Thanks,
Norberto

**Reply by koheinogi** [reply]
That solved the problem, thank you very much.
